libethcore/Exceptions.h

```diff
@@ -48,6 +48,7 @@ DEV_SIMPLE_EXCEPTION( NotEnoughAvailableSpace );
 DEV_SIMPLE_EXCEPTION( NotEnoughCash );
 DEV_SIMPLE_EXCEPTION( GasPriceTooLow );
 DEV_SIMPLE_EXCEPTION( SameNonceAlreadyInQueue );
+DEV_SIMPLE_EXCEPTION( NonceTooMuchInTheFuture );
```

Comment:
What is it for? it is not used anywhere in the code and `geth` doesn't have such exception